core/modules/SConscript

```diff
@@ -115,6 +115,7 @@ def getSrcs(mName):
     return map(lambda s: os.path.join(mName, s), srcs)
 
 def getTgts(mName):
+    print "Calling getTgts(%s)" % mName
```

Comment:
+1 for removing print statement -- now that it's debugged, how often are we going to want to see this?